src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -1028,7 +1246,7 @@ protected override void ProcessRecord()
                             {
                                 _status = ComputerResources.RestartComplete;
                                 WriteProgress(_indicator[indicatorIndex % 4] + _activity, _status, 100, ProgressRecordType.Completed);
-                                _timer.Change(System.Threading.Timeout.Infinite, System.Threading.Timeout.Infinite);
+                                _timer.Change(global::System.Threading.Timeout.Infinite, global::System.Threading.Timeout.Infinite);
```

Comment:
`Win32ShutdownTracker` supports timeout so we could remove the code. /cc @SteveL-MSFT